include/realizations/catchment/Bmi_C_Formulation.hpp

```diff
@@ -135,6 +87,7 @@ namespace realization {
          * @return
          */
         double get_var_value_as_double(const int& index, const std::string& var_name) override;
+        using Bmi_Formulation::get_var_value_as_double;
```

Comment:
Yes, it is (was?) needed, because otherwise the single `override` actually hides the overload set. I was a bit surprised. There may be more going on, or it may have been transient the overloads defined at different layers of the hierarchy as I had changes in progress.

Reply:
For the sake of this cleanup, it seems potentially sensible to eliminate the overload, and just have the handful of call sites that want to assume a scalar value pass an index 0 argument. On the flip side, having them call this overload would let us add a check that the value being requested really is a scalar and not part of a larger array.